create_gazebo_plugins/src/gazebo_ros_create.cpp

```diff
@@ -279,12 +279,12 @@ void GazeboRosCreate::UpdateChild()
   if (set_joints_[LEFT])
   {
     joints_[LEFT]->SetVelocity( 0, wheel_speed_[LEFT] / (wd/2.0) );
-    joints_[LEFT]->SetParam("fmax", 0, torque_);
+    // joints_[LEFT]->SetParam("fmax", 0, torque_);
```

Comment:
We could just take this out instead of commenting. It makes sense that the robot won't move if the max force is zero. Maybe setting this to a more reasonable value would make more sense.